src/pipx/commands/common.py

```diff
@@ -172,3 +178,104 @@ def _get_list_output(
             f"    - {red(name)} (symlink missing or pointing to unexpected location)"
         )
     return "\n".join(output)
+
+
+def _package_name_from_spec(
```

Comment:
This should probably no longer begin with an underscore since we are referencing it from outside of this module.

Reply:
6cda48b